ospf6d/ospf6_top.h

```diff
@@ -38,6 +38,23 @@ enum {
 	OSPF6_LOG_ADJACENCY_DETAIL =	(1 << 1),
 };
 
+struct ospf6_redist {
+	unsigned short instance;
```

Comment:
if we are not using instance why are we storing it?

Reply:
For supporting multiple instances in future, placeholder is provided similar to like current ospfv2 code. 
The API's for redist are written to use instance-id. Therefore we are retaining this parameter.

Reply:
I think we just had this discussion, albeit for a different commit.  save the forward thinking to it's own commit outside of this one.  It is causing confusion.  There is no instance in ospfv3 at this point in time